core/src/main/java/bio/terra/pearl/core/model/portal/Portal.java

```diff
@@ -19,9 +19,11 @@ public class Portal extends BaseEntity {
 
     private String shortcode;
     @Builder.Default
-    private List<PortalParticipantUser> portalParticipantUsers = new ArrayList();
+    private List<PortalParticipantUser> portalParticipantUsers = new ArrayList<>();
     @Builder.Default
     private List<PortalStudy> portalStudies = new ArrayList<>();
     @Builder.Default
     private List<PortalEnvironment> portalEnvironments = new ArrayList<>();
+    @Builder.Default
+    private List<PortalLanguage> portalLanguages = new ArrayList<>();
```

Comment:
these should be attached to the PortalEnvironment, not the portal.  Otherwise there's no way for someone to add a new supported language just in sandbox while it is being developed (and also our general practice is that all configs are environment-isolated, even if they don't need to be)